packages/openapi3-parser/lib/parser/parseMap.js

```diff
@@ -1,11 +1,13 @@
 const R = require('ramda');
 const pipeParseResult = require('../pipeParseResult');
 const {
-  isObject, getValue, isAnnotation,
+  isObject, getValue, isAnnotation, isMember,
 } = require('../predicates');
 const { createWarning } = require('./annotations');
 const parseObject = require('./parseObject');
 
+const isAnnotationOrMember = R.anyPass([isAnnotation, isMember]);
```

Comment:
```suggestion
const isAnnotationOrMember = R.either(isAnnotation, isMember);
```

I'd prefer to use `either` over the more versatile `anyPass` as the implementation is simpler and more efficient. As we only have two items, it is clearer to use `either` semantically.

Reply:
This is a copy from elsewhere in the code :)

Can change all of them in another PR

Reply:
Fair enough, yea I was using `anyPass` and `allPass` a lot before I discovered `either` and `both` 😄

Reply:
https://github.com/apiaryio/api-elements.js/pull/529